app/dashboard/poh_utils.py

```diff
@@ -2,6 +2,15 @@
 
 _poh_contract = None
 
+EIP_1271_ABI = [{"constant":True,"inputs":[{"name":"_messageHash","type":"bytes32"},{"name":"_signature","type":"bytes"}],"name":"isValidSignature","outputs":[{"name":"magicValue","type":"bytes4"}],"payable":False,"stateMutability":"view","type":"function"}]
+
+def is_valid_eip_1271_signature(web3, address, hash, signature) -> bool:
```

Comment:
I'm assuming this is not restricted to just POH so what do you think about moving this over to https://github.com/gitcoinco/web/blob/master/app/grants/utils.py ?

Reply:
Perfect, this was going to be my next question! Other CL will need to call it as well, so this is great timing thanks